packages/ui/src/app/App.stories.tsx

```diff
@@ -497,10 +512,44 @@ export const EmailSubscriptionModalSubscribe: Story = {
     const modal = withinModal(canvasElement)
     const button = modal.getByText(/^Sign and Authorize Email/i)
     expect(button.closest('button')).toBeDisabled()
-    await userEvent.type(modal.getByPlaceholderText('Add email for notifications here'), 'test@email.com')
+    const testEmail = 'test@email.com'
+    await userEvent.type(modal.getByPlaceholderText('Add email for notifications here'), testEmail)
     await waitFor(() => expect(button.closest('button')).toBeEnabled())
+    expect(onSubscribeEmail).toHaveBeenCalledTimes(0)
     await userEvent.click(button)
-    expect(onSubscribeEmail).toHaveBeenCalled()
-    await waitFor(() => expect(modal.getByText('Success!')))
+    await waitFor(
+      () =>
+        expect(onSubscribeEmail).toHaveBeenCalledWith(
+          expect.objectContaining({
+            variables: {
+              id: parseInt(alice.id),
+              name: alice.metadata.name || alice.handle,
+              email: testEmail,
+              signature: expect.any(String),
+              timestamp: expect.any(String),
+            },
```

Comment:
Something goes wrong here from the storybook ui: https://pioneer-2-storybook-git-fork-kdembler-backend-signup-joystream.vercel.app/?path=/story/app--email-subscription-modal-subscribe

Could you have a look please ?